python/ray/dag/compiled_dag_node.py

```diff
@@ -856,7 +856,7 @@ def __init__(
         self._use_default_nccl_group = False
         # This is set to the specified custom nccl group
         # if there exists a type hint of `transport=nccl_group`.
-        self._custom_nccl_group_p2p: Optional[GPUCommunicator] = None
+        self._custom_nccl_group_p2p: Optional[Communicator] = None
```

Comment:
For consistency, it would be good to replace-all `nccl_group` with `communicator`.

Reply:
Do we have to change file name of torch_tensor_nccl_channel.py?

Reply:
Got it. Also we have functions for initializing and destroying `CPUCommunicator` in `torch_tensor_nccl_channel.py`. Do we have to change the file name of `torch_tensor_nccl_channel.py` or to put those functions in a separate file?